app/src/hid_listener.c

```diff
@@ -35,6 +35,15 @@ static int hid_listener_keycode_pressed(const struct zmk_keycode_state_changed *
             return err;
         }
         break;
+    // FIXME: we shouldn't and off the usage page here, we should use the whole
+    // page number
+    case HID_USAGE_PLOVER & 0xFF:
```

Comment:
I don't understand this here. The usage page value should *only* be the plover ID here, so the `& 0xFF` seems unnecessary.

Reply:
Same reason as above